### PR TITLE
update jquery version

### PR DESCRIPTION
**Pull request by bc-bjoern**
## Changes
* Update Version of jquerry from 3.4.1 to 3.6.0

### CODE REVIEW

File: tpl/bootstrap.php [hunk 1]
```diff
@@ -41,7 +41,7 @@
 endif;
 ?>
 		<noscript><link type="text/css" rel="stylesheet" href="css/noscript.css" /></noscript>
-		<script type="text/javascript" data-cfasync="false" src="js/jquery-3.4.1.js" integrity="sha512-bnIvzh6FU75ZKxp0GXLH9bewza/OIw6dLVh9ICg0gogclmYGguQJWl8U30WpbsGTqbIiAwxTsbe76DErLq5EDQ==" crossorigin="anonymous"></script>
+		<script type="text/javascript" data-cfasync="false" src="js/jquery-3.6.0.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
```
Comment (rugk) [inline tpl/bootstrap.php:44]: :heavy_check_mark: verified sri this hash (` sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=`) matches [the one advertised on the jQuery page](https://releases.jquery.com/) for `jquery-3.6.0.min.js`.

**Review by rugk — APPROVED** [review]
If everything works with that, I see of course nothing that would speak against this.

I thought [it was somehow blocked by Bootstrap](https://github.com/PrivateBin/PrivateBin/issues/728), but of course if that's so easy and works, sure, that's great then.
Bootstrap 5 would just remove jQuery altogether. (https://github.com/PrivateBin/PrivateBin/issues/198)

**Comment by bc-bjoern** [comment]
> If everything works with that, I see of course nothing that would speak against this.

For me it seems to work everything. Thanks!

> I thought [it was somehow blocked by Bootstrap](https://github.com/PrivateBin/PrivateBin/issues/728), but of course if that's so easy and works, sure, that's great then. Bootstrap 5 would just remove jQuery altogether. (#198)

I wasn't able to upgrade bootstrap, it's not that easy definitly.

**Comment by elrido** [comment]
Thank you so much! I tested it and it indeed all seems to work. Back in the day I had attempted an upgrade to 3.4.3 which somehow broke the bootstrap checkboxes and had never tried upgrading it since, but this 3.6 version seems to work smoothly.

**Pull request merged**